### PR TITLE
Adding misterFu game object

### PR DESCRIPTION
**Pull request by MIhnatco**
my first ever interaction with ZTM Quest

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "vite": "^5.4.1"
   },
   "dependencies": {
-    "kaplay": "^3001.0.0-beta.1"
+    "kaplay": "^3001.0.0-beta.1",
+    "project": "file:"
```
Comment (r4pt0s) [inline package.json:25]: can you please revert the changes here in the package.json file?

File: src/gameObjects/map_start/misterFu.gameObject.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { scaleFactor } from '../../constants';
+
+export const misterFu = (k, map, spawnpoints) => {
+    k.loadSprite('misterFu', './misterFu.png', {
```
Comment (r4pt0s) [inline src/gameObjects/map_start/misterFu.gameObject.js:4]: unfortunately, there is no misterFu.png available. Please use the characters.png from public instead. The coordinates change in this case as well. Have a look at src/constants.js to get the coordinates for misterFu

File: src/gameObjects/map_start/misterFu.gameObject.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    const misterFuObj = k.make([
+        k.sprite('misterFu', { anim: 'idle-down' }),
+        k.area({
+            shape: new k.Rect(k.vec2(0), 16, 16),
+        }),
+        k.body({ isStatic: true }),
+        k.anchor('center'),
+        k.pos(
+            map.pos.x + spawnpoints.misterFu.x,
```
Comment (r4pt0s) [inline src/gameObjects/map_start/misterFu.gameObject.js:25]: there are no spawnpoints.misterFu.
But you could use player.pos instead which will spawn the character at the same position as the player which will totally work.

File: package-lock.json
Comment (r4pt0s) [inline package-lock.json:1]: please revert the changes here

**Review by r4pt0s — CHANGES_REQUESTED** [review]
Please fix the mentioned comments.
The rest looks good @MIhnatco

File: package.json (hunk 1 above)
Comment (MIhnatco) [reply package.json:25]: I apologize...but I don't have any idea..what it means..to reverte the change?

Comment (r4pt0s) [reply package.json:25]: in your case, just remove 
```json
"project": "file:"
```
and run npm install afterwards which should fix the package-lock.json file as well
@MIhnatco

Comment (r4pt0s) [reply package.json]: Thanks

File: src/gameObjects/map_start/index.js [hunk 4]
```diff
@@ -3,7 +3,13 @@ import { cake } from './cake.gameObject';
 import { npcsOnChairs } from './npcsOnChairs.gameObject';
 import { ztmTrailer } from './ztmTrailer.gameObject';
 import { computer } from './computer.gameObject';
+<<<<<<< HEAD
```
Comment (r4pt0s) [inline src/gameObjects/map_start/index.js:6]: Please remove the merge conflict comments and double check you have all latest changes in this file 🙏🏻

**Review by r4pt0s — CHANGES_REQUESTED** [review]
Just one step and we are good to go 😊

**Review by r4pt0s — APPROVED** [review]
